lib/dns.js

```diff
@@ -126,13 +127,13 @@ function lookup(hostname, options, callback) {
 
   // Parse arguments
   if (hostname && typeof hostname !== 'string') {
-    throw new TypeError('Invalid arguments: ' +
-                        'hostname must be a string or falsey');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'hostname',
+      ['string', 'falsey'], hostname);
```

Comment:
new indentation rule requires aligning to the call `(`
https://github.com/nodejs/node/pull/14403